### ISSUE TITLE
laravel 9 support

### ISSUE THREAD

**Issue by 883** [TerrePorter/StringBladeCompiler#80]
I have been very much helped by this library for some time.
I also want to use string-blade-compiler in my laravel9 project.
Do you have any plans for support?

**Comment by erik-ropez** [comment]
https://github.com/TerrePorter/StringBladeCompiler/pull/81

**Comment by TerrePorter** [comment]
@883 @erik-ropez laravel 9 has built in support for on the fly blade template string rendering. https://laracasts.com/series/whats-new-in-laravel-9/episodes/5

**Comment by erik-ropez** [comment]
> @883 @erik-ropez laravel 9 has built in support for on the fly blade template string rendering. https://laracasts.com/series/whats-new-in-laravel-9/episodes/5

True, but I have a lot of code depending on your `view(['template' => '...'])` syntax. Migrating everything to `Blade::render()` is time consuming.

**Comment by TerrePorter** [comment]
@erik-ropez I see. that would be a lot of work. I have committed your pull request

**Comment by erik-ropez** [comment]
@TerrePorter Could you make a new version release with Laravel 9 support. The last version is https://packagist.org/packages/wpb/string-blade-compiler#6.0.3 and it doesn't support v9.

**Comment by TerrePorter** [comment]
@erik-ropez  I made a new version, 7.0, that is for laravel 9. 

Sorry for taking so long
